src/adr/0034-distributed-listeners-configuration.md

```diff
@@ -161,3 +163,16 @@ The fact that James is failing forces an admin to react, however 'downgraded' st
 believe James is fully operational, while it is not the case.
 
 We did not chose these options as they imply more development for limited safety gains.
+
+## Deploying a new custom listener
+
+Given a new custom listener, not yet deployed in Distributed James cluster,
+
+To be deploying it, an admin needs to follow these steps:
+
+ - Add the jar in extension-jars folder for each James server
+ - As `extension-jars` is read at instanciation time, no reboot is required to instanciate the new listener.
+ - Then the WebAdmin endpoint can be called to configure the given custom listener. The bindings for the new listener 
```

Comment:
you should write this one as a step: "Call the webadmin endpoint ..."
Also, it's not really clear if we pass the configuration in this call, could you make it explicit?

Reply:
> Also, it's not really clear if we pass the configuration in this call, could you make it explicit?

o.O I used the word `configure`

Reply:
> o.O I used the word configure

Sorry but it can be interpreted as "it enables the listener (which can be configured in some file".

Let's remove the ambiguity